the-blue-alliance-ios/View Controllers/MyTBA/MyTBAViewController.swift

```diff
@@ -117,6 +117,7 @@ class MyTBAViewController: ContainerViewController, GIDSignInUIDelegate {
 
             if let error = signOutOperation.completionError {
                 Crashlytics.sharedInstance().recordError(error)
+                self.showErrorAlert(with: "Unable to sign out of myTBA - \(error.localizedDescription)")
```

Comment:
This is related to https://github.com/the-blue-alliance/the-blue-alliance-ios/issues/177#issuecomment-413750734